tools/packaging/packaging.psm1

```diff
@@ -1175,38 +1176,33 @@ function New-MacOSLauncher
     $IsPreview = Test-IsPreview -Version $Version
     $packageId = Get-MacOSPackageId -IsPreview:$IsPreview
 
-    # Define folder for launch application.
-    $macosapp = "$PSScriptRoot/macos/launcher/ROOT/Applications/Powershell.app"
+    # Define folder for launcher application.
+    $suffix = if ($IsPreview) { "-preview" }
+    $macosapp = "$PSScriptRoot/macos/launcher/ROOT/Applications/Powershell$suffix.app"
 
-    # Update icns file.
+    # Create folder structure for launcher application.
+    New-Item -Force -ItemType Directory -Path "$macosapp/Contents/MacOS" | Out-Null
+    New-Item -Force -ItemType Directory -Path "$macosapp/Contents/Resources" | Out-Null
+
+    # Define icns file information.
     $iconfile = "$PSScriptRoot/../../assets/Powershell.icns"
     $iconfilebase = (Get-Item -Path $iconfile).BaseName
 
-    # Create Resources folder, ignore error if exists.
-    New-Item -Force -ItemType Directory -Path "$macosapp/Contents/Resources" | Out-Null
+    # Copy icns file.
     Copy-Item -Force -Path $iconfile -Destination "$macosapp/Contents/Resources"
 
-    # Set values in plist.
+    # Create plist file.
     $plist = "$macosapp/Contents/Info.plist"
-    Start-NativeExecution {
-        defaults write $plist CFBundleIdentifier $packageId
-        defaults write $plist CFBundleVersion $Version
-        defaults write $plist CFBundleShortVersionString $Version
-        defaults write $plist CFBundleGetInfoString $Version
-        defaults write $plist CFBundleIconFile $iconfilebase
-    }
-
-    # Convert to XML plist, needed because defaults native
-    # app auto converts it to binary format when it modify
-    # the plist file.
-    Start-NativeExecution {
-        plutil -convert xml1 $plist
-    }
+    $plistcontent = $packagingStrings.MacOSLauncherPlistTemplate -f $packageId, $Version, $iconfilebase
```

Comment:
Did you forget to include this change to the strings?

Reply:
This has been corrected.